src/Import/src/FMI3/fmi3_import_unit.c

```diff
@@ -81,39 +81,47 @@ double fmi3_import_get_display_unit_offset(fmi3_import_display_unit_t* du) {
     return fmi3_xml_get_display_unit_offset(du);
 }
 
-fmi3_float64_t fmi3_import_float64_convert_to_display_unit(fmi3_float64_t val , fmi3_import_display_unit_t* du, int isRelativeQuantity) {
+int fmi3_import_get_display_unit_inverse(fmi3_import_display_unit_t* du) {
+    return (int)fmi3_xml_get_display_unit_inverse(du);
+}
+
+fmi3_float64_t fmi3_import_float64_convert_to_display_unit(fmi3_float64_t val, fmi3_import_display_unit_t* du, int isRelativeQuantity) {
     double factor = fmi3_import_get_display_unit_factor(du);
-    double offset = fmi3_import_get_display_unit_offset(du);
-    if (isRelativeQuantity)
-        return val * factor;
-    else
-        return (val*factor + offset);
+    if (fmi3_import_get_display_unit_inverse(du)) { // inverse="true" requires offset=0
+        return factor / val;
+    } else {
+        double offset = isRelativeQuantity ? 0.0 : fmi3_import_get_display_unit_offset(du);
+        return factor * val + offset;
+    }
 }
 
 fmi3_float64_t fmi3_import_float64_convert_from_display_unit(fmi3_float64_t val, fmi3_import_display_unit_t* du, int isRelativeQuantity) {
```

Comment:
I actually noticed that we are somewhat inconsistent here, typically in the `*import_*.c` functions we typically just forward the call to the respective `*_xml*.c/h` function, but not here. There are corresponding `*_xml*.c/h` functions for these conversions here, but they are unused. 

Forward the calls for consistency or removed the unused ones? @filip-stenstrom  Maybe you also have thoughts here.

Reply:
I'd calculate it in _xml.c. This is a sort of convenience method, so could be in _import only, but for now I think _xml is more consistent.

Reply:
Fixed.